resources/views/store/voucher_allocation.blade.php

```diff
@@ -0,0 +1,152 @@
+@extends('store.layouts.service_master')
+
+@section('title', 'Voucher Collection')
+
+@section('content')
+
+    @include('store.partials.navbar', ['headerTitle' => 'Voucher Collection'])
+
+    <div class="content">
+        <form>
+            <div class="col">
+                <div>
+                    <img src="{{ asset('store/assets/info-light.svg') }}">
+                    <h2>This Family</h2>
+                </div>
+                <div class="alongside-container">
```

Comment:
This is intentionally hardcoded for the testing session?

Has it been carded to fill with dynamic values? So we don't forget.